src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$title-color: #060b35;
+$paragraph-color: #616070;
+$white-color: #fff;
+$blue-color: #00acdc;
```

Comment:
You need to fix the variable names, they should show their purpose, not their content. if I decided to change the color, I would have to change it everywhere, not just in the variable definition.